### PR TITLE
Turn some static variables into static stored constants

### PR DESCRIPTION
**Pull request by rintaro**
Computing `OperatorTable.standardOperators` is not trivial, make it `static let` instead of computed static variable.
So as `SyntaxProtocol.structure` implementations.

### CODE REVIEW

**Comment by rintaro** [comment]
@swift-ci Please test

File: Sources/SwiftSyntax/SyntaxProtocol.swift [hunk 1]
```diff
@@ -225,13 +225,10 @@ extension SyntaxProtocol {
   }
 
   public var keyPathInParent: AnyKeyPath? {
-    guard let parent = self.parent else {
-      return nil
-    }
-    guard case .layout(let childrenKeyPaths) = parent.kind.syntaxNodeType.structure else {
+    guard case .layout(let childrenKeyPaths) = self._syntaxNode.parent?.kind.syntaxNodeType.structure else {
```
Comment (ahoppen) [inline Sources/SwiftSyntax/SyntaxProtocol.swift:228]: Is `self._syntaxNode.parent` faster than `parent`?

**Review by ahoppen — APPROVED** [review]

File: Sources/SwiftSyntax/SyntaxProtocol.swift (hunk 1 above)
Comment (rintaro) [reply Sources/SwiftSyntax/SyntaxProtocol.swift:228]: That was my measurement, but I'm not 100% sure tbh. I will double check.

Comment (rintaro) [reply Sources/SwiftSyntax/SyntaxProtocol.swift]: Removed this commit from the PR for now

**Comment by rintaro** [comment]
@swift-ci Please test

**Comment by rintaro** [comment]
@swift-ci  Please test Windows

**Comment by rintaro** [comment]
@swift-ci Please test Windows

**Pull request merged**